federation-js/src/service/printComposedSdl.ts

```diff
@@ -144,18 +142,17 @@ function printSchemaDefinition(
   }
 
   return (
-    'schema' +
-    // Federation change: print @graph and @composedGraph schema directives
-    printFederationSchemaDirectives(serviceList) +
+    'schema @using(spec: "https://specs.apollo.dev/cs/v0.1")' +
     `\n{\n${operationTypes.join('\n')}\n}`
   );
 }
 
-function printFederationSchemaDirectives(serviceList: ServiceDefinition[]) {
-  return (
-    serviceList.map(service => `\n  @graph(name: "${service.name}", url: "${service.url}")`).join('') +
-    `\n  @composedGraph(version: 1)`
-  );
+function printGraphs(serviceList: ServiceDefinition[]) {
+  return `enum cs__Graph {${
+    serviceList.map(service =>
+      `\n  ${service.name} @cs__link(to: { http: { url: ${JSON.stringify(service.url)} } })`
```

Comment:
Isn't `service.url` just a `string`? Do we need to stringify it?

Reply:
if it needs escapes, `JSON.stringify` will provide those